lib/yandex_cleanweb/version.rb

```diff
@@ -1,3 +1,3 @@
 module YandexCleanweb
-  VERSION = "0.0.6"
+  VERSION = "0.0.7"
```

Comment:
I will bump the version in a separate commit. Can you remove this change, please?